example/zip/linestring.js

```diff
@@ -0,0 +1,28 @@
+/*
+node example/zip/linestring.js
+*/
+var zip = require('../../src/zip'),
+  fs = require('fs');
+
+var linestringGeojson = require('../../test/geojson/LineString-2d-single.json');
+
+var zipOptions = {
+  types: {
+    polyline: 'polyline',
+  },
+};
+
+zip(linestringGeojson, zipOptions).then((base64String) => {
+  fs.writeFile(
+    'test_linestring.shp.zip',
+    base64String,
+    { encoding: 'base64' },
+    function (err) {
+      if (err) {
+        console.log('Error', err);
+      } else {
+        console.log('File created');
+      }
+    },
+  );
+});
```

Comment:
A utility function for to handle writing the zip file might have been useful for all of these examples.

```js
// utility
function createWriteZipFunction(fileName) {
  return function(base64String) {
    fs.writeFile(
      fileName,
      base64String,
      { encoding: 'base64' },
      function (err) {
        if (err) {
          console.log('Error', err);
        } else {
          console.log('File created');
        }
      },
    );
  }
}

// use the utility everywhere
var writeZip = createWriteZipFunction('test_linestring.shp.zip');

zip(linestringGeojson, zipOptions).then(writeZip);
```